lib/card_scroll_widget.dart

```diff
@@ -9,11 +10,16 @@ class CardScrollWidget extends StatelessWidget {
   final double currentPage;
   final double padding = 20.0;
   final double verticalInset = 20.0;
+ 
+  
 
   CardScrollWidget(this.currentPage);
-
   @override
   Widget build(BuildContext context) {
+    //写得不太好这种方法虽然方便
```

Comment:
Please use English all the time for commits.

Reply:
Okay Okay!